shortages-prediction-adapters/src/test/groovy/e2e/E2eSpec.groovy

```diff
@@ -0,0 +1,41 @@
+package e2e
+
+import java.util.concurrent.TimeUnit
+
+import org.awaitility.Awaitility
+import org.junit.runner.RunWith
+import spock.lang.Specification
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration
+import org.springframework.boot.autoconfigure.context.PropertyPlaceholderAutoConfiguration
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.web.client.TestRestTemplate
+import org.springframework.http.ResponseEntity
+import org.springframework.test.context.junit4.SpringRunner
+
+import static org.assertj.core.api.BDDAssertions.then
+
+/**
+ * @author Marcin Grzejszczak
+ */
+@SpringBootTest(classes = E2eSpec.class,
+		webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ImportAutoConfiguration(PropertyPlaceholderAutoConfiguration)
+class E2eSpec extends Specification {
+
+	@Value('${application.url}') String applicationUrl
+	@Value('${test.timeout:60}') Long timeout
+
+	TestRestTemplate testRestTemplate = new TestRestTemplate();
+
+	def 'should work'() {
+		expect:
+			Awaitility.await().atMost(this.timeout, TimeUnit.SECONDS).untilAsserted({
```

Comment:
Why you use awaitility when in Spock you have PollingConditions?

Reply:
Because I copied it from a working JUnit test. Feel free to change it to whatever you want.